packages/fxa-settings/src/pages/Signin/index.test.tsx

```diff
@@ -101,6 +109,23 @@ describe('Signin', () => {
     expect(signinHeader).toBeInTheDocument();
     expect(pocketLogo).toBeInTheDocument();
     expect(passwordInputForm).not.toBeInTheDocument();
+
+    // Pocket links should always open in a new window (announced by screen readers)
```

Comment:
Oh, that's nice! For the Pocket links though I might keep it this way because this differentiates these links from the Firefox links that don't open in a new window.